### PR TITLE
Upgrade to Livewire v3

### PR DESCRIPTION
**Pull request by r2luna**
## PR Title

Upgrade to work with Livewire v3

### Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by marchershey** [comment]
Hopefully @usernotnull looks at this soon. It's the only package that is holding me back from upgrading lol

**Comment by usernotnull** [comment]
I'm alive :)
Will merge this within a week, kindly test it in the meantime and let me know

**Comment by marchershey** [comment]
@usernotnull Can you push this PR into a dev branch?

File: README.md [hunk 1]
```diff
@@ -207,43 +207,37 @@ Next, you need to register `Toast` with AlpineJS.  How this is done depends on w
...
 ```
 
-#### AlpineJS added via script tag
-
-If you imported AlpineJS via a script tag simply add the `@toastScripts` blade directive *BEFORE* importing AlpineJS:
+```js
+import {Alpine, Livewire} from '../../vendor/livewire/livewire/dist/livewire.esm';
+import ToastComponent from '../../vendor/usernotnull/tall-toasts/resources/js/tall-toasts'
 
+Alpine.data(ToastComponent)
```
Comment (marchershey) [inline README.md:235]: This should be `Alpine.plugin(ToastComponent)`

**Pull request merged**